src/blog-utils/StaticAddonCard/index.js

```diff
@@ -35,19 +45,37 @@ const StaticAddonCard = ({ addon }: Props): React.Node => {
 
       <AddonBadges addon={addon} />
 
-      <p
-        className="AddonSummary"
-        // eslint-disable-next-line react/no-danger
-        dangerouslySetInnerHTML={sanitizeHTML(nl2br(summary), ['a', 'br'])}
-      />
-
-      <GetFirefoxButton
-        addon={addon}
-        buttonType={GET_FIREFOX_BUTTON_TYPE_ADDON}
-        className="AddonFirefoxButton"
-      />
+      <div className="AddonSummary">
```

Comment:
Are we not sticking with our usual convention for classNames for this component (e.g., `StaticAddonCard-AddonSummary`)?

Reply:
we should 😉 I'll update the class names.